### PR TITLE
Update Expect to Fix Vulnerability

### PR DESCRIPTION
**Pull request by matthew1534**
Upgraded Expect

to fix Regular Expression Denial of Service in `expect > jest-message-util > micromatch > braces` versions <2.3.1
See report here: https://www.npmjs.com/advisories/786

I don't believe this version change negatively effects anything in this package

- [Y ] Unit tests
- [ Y] Documentation is up to date
- [Y ] No additional lint warnings
- [Y ] Add yourself to contributors list (`yarn contributor`)
- [Y ] [Typescript definitions](https://github.com/jest-community/jest-extended/blob/master/types/index.d.ts) are added/updated where relevant

### CODE REVIEW

**Comment by iilei** [comment]
can you release it as a canary release maybe? That would also allow for feedback rgd breaking changes

**Comment by mattphillips** [comment]
LGTM thanks @matthew1534 for the PR and @benjaminkay93 for reviewing!

**Comment by matthew1534** [comment]
Amazing - thanks @mattphillips 👍

**Comment by milgner** [comment]
Nice! Do you have plans to release a new version with the fix included?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Head over to [here](https://hacktoberfest.digitalocean.com/sign_up/register) to
  - [`jest-matchers-utils`](https://github.com/facebook/jest/tree/master/packages/jest-matcher-utils) is being used for syntax highlighting of error messages.
    * See the Jest docs for an [example usage](https://facebook.github.io/jest/docs/en/expect.html#thisutils)
  - Jest's [`expect`](https://github.com/facebook/jest/tree/master/packages/expect) package is being used to access their deep `equals` function.
-   * `import { equals } from 'expect/build/jasmine_utils';`
+   * `import { equals } from 'expect/build/jasmineUtils';`
```
Comment (SimenB) [inline CONTRIBUTING.md]: why? it's injected as `this.equals` in custom matchers - please don't reach into private files

https://jestjs.io/docs/en/expect#thisequalsa-b

Comment (SimenB) [reply CONTRIBUTING.md]: I just came here from the release, I know it's not really related to this PR (although this change is a good example of why it's a bad idea)

**Pull request merged**